README.md

```diff
@@ -156,6 +160,12 @@ COMMANDS:
      signup     Create your Scalingo account
      self       Get the logged in profile
      whoami     Get the logged in profile
+     
+   Integrations:
```

Comment:
What are these modifications? Can't you make a PR with only the repo link commands?

Reply:
I have merged #444 in this branch to reuse some functions integrated into #444